bot/exts/halloween/spookynamerate.py

```diff
@@ -195,7 +195,7 @@ async def delete_name(self, ctx: Context) -> None:
     @Cog.listener()
     async def on_reaction_add(self, reaction: Reaction, user: User) -> None:
         """Ensures that each user adds maximum one reaction."""
-        if not await self.messages.contains(reaction.message.id) or user == self.bot.user:
+        if not await self.messages.contains(reaction.message.id) or user.bot:
```

Comment:
```suggestion
        if user.bot or not await self.messages.contains(reaction.message.id):
```
Much friendlier to check for bot first, then check the cache if it's not a bot.

Reply:
But won't users react to messages more often than a bot?

Reply:
But we can avoid check from Redis when user is bot.

Reply:
Yeah that. The redis cache check is a lot heavier than a simple bool check.

Reply:
Hmm ok then